---
layout: post
title: About Acidithiobacilli taxonomy
date: 2024-06-04 11:57:00
description: a summary of papers i'm currently reading as part of the Microbial Ecophisiology Lab
tags: visit, chile, microbial ecophisiology lab
categories: paper reading
---

As part of my visit/internship you could say? I don't know, but I will be reading some papers regarding Acidithiobacilli taxonomy and classification. This is because I'll be working on this for the next 6 months.

So, for starters, I do know about extremophiles, but this will be my first time working with them and their genomics. This is of course an exciting opportunity for doing some astrobiology after years of just babbling about it.

As a way to start doing some academia-related posts in this personal page, I'll be doing my first post on these readings. And I'll probably keep writing this stuff since it is a great way to remember and just get into the topic.

Ok, here come the papers:

#### A new view of the tree of life (Hug et. al. 2016)

So, this one is attempting to incorporate new criteria for the reconstruction of the tree of life. Why? Well, apparently with the current most beloved method (the small subunit ribosomal RNA), certain perspectives are lost to the untrained eye (like mine). Some questions regarding the diversity and structure of the tree remained (which makes me think what is a structre of a tree exactly? Are we only talking about the topology of it?). What the paper is arguing for is that the genome has a lot more to offer than we might think of, so it is natural to just step out of the standard approach with 16S.

The group has built a new tree of life making use of all genomes recovered from public repositories and 1011 newly reconstructed genomes that they recovered. From these genomes, they aligned and concatenated 16 ribosomal protein sequences from each organism. This is advantegous because these genes actually have a function, which means that there would be no artifacts from untranslated genes that might be under different evolutionary forces. I guess this is advantegous in the sense that evolution mainly affects the phenotype rather than the genotype. I mean, selection mainly works on the phenotype, right? Genotype is a derivatve of this pressure on the phenotype.

Another advantage is that they tend to be syntenic (which would allow it to be easily identifiable, right? Since it can be easily aligned, right? Idk, I'm trying to think over here) and they can be co-located (?) in a small genomi region in Bacteria and Archaea, reducing binning errors (?) that could perturb the geometry of the tree.